src/OSPSuite.UI/Views/Importer/ImportConfirmationView.cs

```diff
@@ -26,6 +28,21 @@ public string SelectedSeparator
          get => separatorComboBoxEdit.SelectedText;
       }
 
+      public void DisableSelectingDataSets(string errorMessage)
```

Comment:
I don't like this. You are mixing two properties and that's why you are duplicating the call (look the first 3 lines are the same in the Enable and Disable)

You may need to methods, but it should be " bool SelectingDataSetsEnable {set}" (potentially get but you probably don't need it aside from tests)

And then you need a method "SetErrorMessage(string message)"

Reply:
@msevestre  I like your suggestion way better! But then you mean listen to the propertyChanged to actually set the properties? If at some point you get 5 min time let's discuss this on a call

Reply:
No. I'll hope on a call with you later today

Reply:
@msevestre call anytime :)

Reply:
I am on slack.